app/views/course/material/folders/show.html.slim

```diff
@@ -1,4 +1,4 @@
-= page_header(@folder.name)
+= page_header(display_name(@folder))
```

Comment:
link to all files? I don't know.

The name of the function is called `display_user` and `link_to_user`